assets/src/edit-story/karma/fixture/containers/designPanel/colorPreset.js

```diff
@@ -28,16 +28,28 @@ export class ColorPreset extends AbstractPanel {
     super(node, path);
   }
 
-  get add() {
-    return this.getByRole('button', { name: /Add color preset/ });
+  get addGlobal() {
+    return this.getByRole('button', { name: /Add global color/ });
   }
 
-  get apply() {
-    return this.getByRole('button', { name: /Apply color preset/ });
+  get addLocal() {
+    return this.getByRole('button', { name: /Add local color/ });
   }
 
-  get delete() {
-    return this.getByRole('button', { name: /Delete color preset/ });
+  get applyGlobal() {
```

Comment:
Hmm, I suppose this could be something like `apply(type)` instead and then return `this.getByRole('button', { name: 'Delete ${type} preset' });` instead buf not sure if would be an actual improvement in terms of usability and potential typos.

Reply:
`.colorPreset.local.apply` and `.colorPreset.global.apply` perhaps?

```js
class ColorPresetType extends Container {
  constructor(node, path) {
    super(node, path);
    this._type = path;
  }
  get add() {
    return this.getByRole('button', { name: `Add ${this._type} color` });
  }
  get apply() {
    return this.getByRole('button', { name: `Apply ${this._type} color` });
  }
  get delete() {
    return this.getByRole('button', { name: `Delete ${this._type} color` });
  }
}
export class ColorPreset extends AbstractPanel {
  constructor(node, path) {
    super(node, path);
  }
  get local() {
    return this._get(this.node, 'local', ColorPresetType);
  }
  get global() {
    return this._get(this.node, 'global', ColorPresetType);
  }
}
```